bluesky_browser/frameworks/qt/search.py

```diff
@@ -209,11 +210,9 @@ def show_results(self):
         self.show_results_event.clear()
         t0 = time.monotonic()
         counter = 0
```

Comment:
Do we need the `counter` here as its corresponding update has been removed?

The code on line 230 may need to be cleaned as well.

Reply:
Oh, yes, good point. My proposal breaks that logging at the end of show_results(). I put the increment back in at https://github.com/bluesky/bluesky-browser/pull/57/commits/cc4d39455892847dd2af9deb5d4afba91f9acfb8

Reply:
Cool, thanks!